README.md

```diff
@@ -30,7 +30,7 @@ Once installed you need to import our main module:
 ```js
 import {NgbModule} from '@ng-bootstrap/ng-bootstrap';
 ```
-The only remaining part is to list the imported module in your application module. The exact method will be slightly
```

Comment:
This changes the meaning of this sentence, really. What we are talking about here is the main module of an application.